likeminds-chat-reactnative-integration/ChatSX/store/actions/chatroom.ts

```diff
@@ -23,15 +23,13 @@ import {
 } from "../types/types";
 import { Client } from "../../client";
 
-const myClient = Client.myClient;
-
 export const getConversations =
```

Comment:
chatroom is a file containing methods which are exported from there, so if we declare `const myClient = Client.myClient;` globally, flow will never go there since its not a component and hence `myClient` inside methods will be undefined